sympy/core/symbol.py

```diff
@@ -201,15 +201,43 @@ def numbered_string_incr(s, start=0):
 
 class Symbol(AtomicExpr, Boolean):
     """
+    Symbols are variables, and like all other variables, they need to
+    be assigned before you can use them.
+    SymPy :func:`Symbol` function returns just a single symbol taken from argument ``name`` whereas :func:`symbols` function can be used
+    to return a sequence of symbols.
+    Symbols can be used to build SymPy expressions which are immutable.
+
+    Parameters:
+    ===========
+
+    name: a string
+
     Assumptions:
-       commutative = True
+    ============
 
-    You can override the default assumptions in the constructor.
+    commutative = True
 
     Examples
     ========
 
-    >>> from sympy import symbols
+    >>> from sympy import Symbol, symbols
+    >>> a = Symbol('a')
+    >>> x, y, z = symbols('x y z')
+
+    SymPy will automatically pretty print symbols with greek letters.
+
+    >>> alpha, beta, gamma = symbols('alpha beta gamma')
+    >>> 2*alpha + 3*beta - gamma #doctest: +SKIP
+    2α + 3β - γ
+
+    Sympy will automatically pretty print symbols with subscripts.
+
+    >>> alpha1, alpha2 = symbols('alpha1 alpha2')
+    >>> 2*alpha1 + 3*alpha2 #doctest: +SKIP
+    2α₁ + 3α₂
+
+    You can override the default assumption in the constructor.
+
     >>> A,B = symbols('A,B', commutative = False)
```

Comment:
Commutative is one of the least useful properties to set.

It would be good to show some quick examples of more useful things like `positive=True` here and then link to this page for more information:
https://docs.sympy.org/latest/guides/assumptions.html

Reply:
Note that `even=False` is a bad example as it's almost always not going to do anything useful:
https://docs.sympy.org/latest/guides/assumptions.html#interpretation-of-the-predicates

Reply:
@oscarbenjamin My bad ,I opened the doc and that was the first thing I got my eyes on ! It might not be an important assumption to document/add here !

Reply:
Thank you for the review and response Sir @oscarbenjamin  and @faze-geek I just implemented the changes, can you please check once are they okay or should I add something more

Reply:
flake8 check is failing I think due to the use of `sqrt`  is there something I can do for that to pass that check?

Reply:
You have missed that .Otherwise I still think the documentation is partial but @oscarbenjamin could comment better on that !